README.md

```diff
@@ -86,20 +84,124 @@ code because of all the basic column-renamings and the full listing of health co
 side, we've been told by DAP leadership that the HLES structure is effectively immutable on their end,
 so it's unlikely we'll ever need to rewrite large chunks of this processing logic.
 
-### Hack: Uploading Results to Terra
-The TDR will eventually be fully integrated with Terra Workspaces. In the immediate future, however,
-there's no "golden path" for bridging data from the repository into the workspace model. DAP's near-term
-users aren't very technical, so it's unlikely they'll be comfortable figuring things out on their own.
+## Additional Datasets
 
-To give ourselves some breathing room, we've written a [script](./hack/convert-output-to-tsv.py) to convert
-the outputs of our transformation pipeline into workspace-compatible TSVs. We can manually run the script
-after the two Dataflow pipelines as a final transformation step, then upload the outputs to a workspace.
+### CSLB Data
+The Canine Social and Learned Behavior is a followup survey to HLES that is administered annually. 
+The goal of this survey is to assess age-related cognitive and behavioral changes in dogs. The first 
+time a participant fills out the CSLB survey, a baseline score for  is established. Participants will 
+have the opportunity to complete the survey again every year. This repeated design allows us to learn 
+how the dogs in the study change over time. Answers to all questions are required.
+
+#### CSLB Extraction
+* Pull all records from each yearly arm *"annual_{yyyy}_arm_1"*
+* _canine_social_and_learned_behavior_complete_ is marked as complete
+
+#### CSLB Schema Design
+* A single `cslb` table which contains a single CSLB record per dog per year. 
+* The table includes the 16 CSLB question responses and a foreign key to the dog table. 
+
+
+## Environmental Data
+Environmental Data in the Dog Aging Project is published monthly to RedCap for each dog using the addresses
+collected in the HLES survey. It seeks to capture a snapshot of key environmental factors that may affect a 
+dog's health over time so it can be used in conjunction with the other datasets. All fields from environmental 
+data are calculated using the HLES primary and secondary addresses.
+
+#### Environmental Extraction
+* Pull all records from each monthly arm for each address: *"annual_{MMMyyyy}_arm_1"*, *"annual_{MMMyyyy}_secondary_arm_1* 
+* _baseline_complete_ is marked as complete
+
+#### Environment Schema Design
+The Environmental data was pretty straightforward as DAP provided us with a proto-schema which we
+converted to a repo schema. We decided to break the larger table up into smaller fragments: 
+_geocoding, census, pollutants, temperature_precipitation, and walkability_ variables.
+
+* A single `environment` table which contains a single Environment record per dog per month per address. 
+* The table includes 147 environmental variables and a foreign key to the dog table. 
```

Comment:
⛏️ I suggest avoiding the specific # of environmental variables here as that can get out of date quickly:
```suggestion
* The table includes all environmental variables and a foreign key to the dog table. 
```